js/modules/types/message.js

```diff
@@ -353,7 +234,7 @@ const toVersion5 = exports._withSchemaVersion(5, initializeAttachmentMetadata);
 const toVersion6 = exports._withSchemaVersion(
   6,
   exports._mapContact(
-    exports._cleanAndWriteContactAvatar(Attachment.migrateDataToFileSystem)
+    Contact.parseAndWriteContactAvatar(Attachment.migrateDataToFileSystem)
```

Comment:
Super minor: Now that you have `Contact.*` module, you could shorten function to `Contact.parseAndWriteAvatar` since it’s clear the avatar is in reference to a `Contact`. Seeing these noun infixes (`*Contact*`) is what often nudges me to create a new module 😄